Services/Ordering/Ordering.Application/Commands/CreateOrder/CreateOrderUseCase.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Ordering.Application.Repositories;
+using Ordering.Domain.Orders;
+using Ordering.Domain.ValueObjects;
+
+namespace Ordering.Application.Commands.CreateOrder
+{
+    public class CreateOrderUseCase : ICreateOrderUseCase
+    {
+        private readonly IOrderWriteOnlyRepository _orderWriteOnlyRepository;
+
+        public CreateOrderUseCase(IOrderWriteOnlyRepository orderWriteOnlyRepository)
+        {
+            _orderWriteOnlyRepository = orderWriteOnlyRepository;
+        }
+
+        public async Task<CreateOrderResult> Execute(string street, string city, string state, string country, string zipCode)
```

Comment:
it is a good practice to use as least arguments in method as possible. so, in your case it could be just Adress or Order